views/templates/block/channelInfo.twig

```diff
@@ -1,47 +1,219 @@
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Version number:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <span class="version">{{ upgradeInfo.version_num }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.link is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'URL:'|trans }}</label>
-                <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'URL:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.md5 is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
-                <span class="md5">{{ upgradeInfo.md5 }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <span class="md5">{{ upgradeInfo.md5 }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.changelog is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Changelog:'|trans }}</label>
-                <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Changelog:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+                </div>
             </div>
         {% endif %}
+        <div class="row">
+            <div class="col-xs-3 col-md-1">
+                <label class="label-small">{{ 'PHP compatibility:'|trans }}</label>
+            </div>
+            <div class="col-xs-9 col-md-11">
+                <span class="phpcompatibility">{{ 'Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.
+                    For example, if your current version is 1.6.1.x and you want to upgrade to 1.7.8, only the PHP 7.1 would be compatible with both.'|trans }}
+                </span>
+                <table class="table-striped">
+                <thead>
```

Comment:
@matthieu-rolland I'm pretty sure you can render such table based on the compatibility range array

Reply:
I'm applying the specs, the dynamic table is coming later according to the plan established by PMs

Reply:
I agree with @kpodemski though, since some tool was added to detect compatibility a little refacto would allow to build a compatibility matrix easily and make all this table more dynamic and easier to maintain in the future (see my comment in `UpgradeSelfCheck` class